index.html

```diff
@@ -0,0 +1,206 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+    integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div class="after-bdy">
+    <div class="portfolio-class">
+      <nav class="flex white-back">
+        <a id="title" href="#">Logo</a>
+        <div class="flex icon-stack-container">
+          <span id="icon-stack"></span>
+        </div>
+      </nav>
+      <main class="main-headline white-back">
+        <article>
+          <h1>
+            I'm Ahmed<br />
+            Glad to see <br />
+            you!
+          </h1>
+          <p>
+            I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <div class="socials">
+            <h6>LET'S CONNECT</h6>
+            <ul class="icons_social__container flex">
```

Comment:
These icons also have `font-size: 20px` in Figma Design. Kindly correct this too.
![image](https://user-images.githubusercontent.com/61361037/219710939-a23f3ca1-ce11-426d-a2be-203325ee75ed.png)